src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2840,7 +2911,9 @@
         - name: Gets all the available BGP service communities.
           text: az network route-filter rule list-service-communities -o table
         - name: Get the community value for Exchange.
-          text: az network route-filter rule list-service-communities --query '[].bgpCommunities[?communityName==`Exchange`].[communityValue][][]' -o tsv
+          text: |
+            az network route-filter rule list-service-communities\\
+            --query '[].bgpCommunities[?communityName==`Exchange`].[communityValue][][]' -o tsv
```

Comment:
Indent continuation lines for all examples and put space before `\\` continuation

Reply:
ok, went through all of the examples, found/fixed all examples with missing space before \\ and indentation. Cleaned up some others in there too.